admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl

```diff
@@ -28,17 +28,26 @@
     <i class="material-icons">account_circle</i>
   </div>
   <div class="dropdown-menu dropdown-menu-right">
-    <div class="text-center employee_avatar">
-      <img class="avatar rounded-circle" src="{$employee->getImage()}" />
-      <span>{$employee->firstname} {$employee->lastname}</span>
-    </div>
-    <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
-      <i class="material-icons">settings_applications</i>
-      {l s='Your profile'}
+    <div class="employee-wrapper-avatar">
+      
+      <span class="employee_avatar"><img class="avatar rounded-circle" src="{$employee->getImage()}" /></span>
+      <span class="employee_profile">{l s='Welcome back'} {$employee->firstname}</span>
```

Comment:
I think it would be better to use sprintf:

```smarty
{l s='Welcome back %name%' sprintf=['%name%' => $employee->firstname] d='Some.Domain.Here'}
```

Of course, the real domain will have to be defined by @LouiseBonnard

Reply:
I changed it to your example